src/expression.rs

```diff
@@ -13,10 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
-use crate::{imag, instruction::MemoryReference, real};
 use std::collections::HashMap;
 use std::f64::consts::PI;
 use std::fmt;
+use std::str::FromStr;
+
+use crate::parser::{lex, parse_expression};
+use crate::{imag, instruction::MemoryReference, real};
```

Comment:
👍 Reminds me of the `flake-import-order` setup for Python: standard imports before third-party before local. I wonder if we can turn this into a `clippy` lint?

Reply:
JetBrains IDEs do that for you 😁. Looks like rustfmt can be configured to do it, just doesn't by default, search for "group_imports": https://rust-lang.github.io/rustfmt/